devops/tartarus-helm/Chart.yaml

```
@@ -0,0 +1,17 @@
+type: application
+
+version: 1.0.0
+appVersion: 1.0.0
+
+dependencies:
+- name: postgresql
+  version: "16.3.5"
+  repository: "https://charts.bitnami.com/bitnami"
+  condition: postgresql.enabled
+  tags:
+    - tartarus
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)